crl-test/outputs.tf

```
@@ -4,6 +4,11 @@
 # output service accounts.
 #
 
-output "service_account_email" {
-  value = google_service_account.crl.email
+output "service_account_email-admin" {
```

I don't think we should mix underscores and dashes in resource names. The current standard is underscores so I suggest we stick with that. The SA id can have dashes but all Terraform names should have underscores.
```suggestion
output "service_account_email_admin" {
```